challenges/binary_search/binary_search.py

```diff
@@ -0,0 +1,17 @@
+def binary_search(arr, val):
+    first = 0; last = len(arr)-1
```

Comment:
If you are defining multiple variables on one line, it is more pythonic to do it like this
```python
first, last = 0, len(arr) - 1
```
Or just do it on multiple lines.